pyproject.toml

```diff
@@ -231,7 +231,7 @@ filterwarnings = [
   # treat warnings as errors
   "error",
   # until there is a new dateutil release: github.com/dateutil/dateutil/pull/1285
-  "ignore:datetime.datetime.utc:DeprecationWarning",
+  # "ignore:datetime.datetime.utc:DeprecationWarning",
```

Comment:
If commenting this out works, you can just remove this line and the line above

Reply:
I tried for testing since that is still a draft PR but indeed will delete.